formats/definitions.json

```diff
@@ -142,6 +142,54 @@
         }
       }
     },
+    "nodes": {
```

Comment:
Does this need to be a definition if it's only used in simple smart answers?

This data structure is complex and needs some description fields to explain it.